day_6/exercises/burrito.js

```diff
@@ -20,8 +20,14 @@ class Burrito {
 
 var dinner = new Burrito("Beans", "Rice", ["cheese", "salsa", "guacamole"]);
 
-// Call the methods here
+dinner.toppings.push("avacado");
+dinner.toppings.shift();
+dinner.protein = "Mole";
+
 
 console.log(dinner.protein);
 console.log(dinner.base);
 console.log(dinner.toppings);
+
+// 6/7/2020: I came back to look at this and I think the methods need to be
+// with in the curly brackets for the class. This way does work? Is it acceptable? 
```

Comment:
For future reference, the direction `add the following methods to the Burrito class` means you're defining methods like you did on `person.js` and `dog.js`, so yes, those methods would be inside the curly brackets of Burrito.